scopes/component/ui/compare/component-compare.module.scss

```diff
@@ -52,7 +52,15 @@
 }
 
 .tab {
+  // TODO - create z-index pallete system.
+  z-index: 10; // to prevent it from hiding behind aspect tab content.
```

Comment:
should avoid using arbitrary numbers and if we have to use a value that doesn't exist in `~@teambit/ui-foundation.ui.constants.z-indexes/z-indexes.module.scss`
then we should add it and reference from their
Otherwise when we have to clean up its going to take a while to figure out all the places we use arbitrary z-index values